gammapy/maps/wcs.py

```diff
@@ -400,8 +402,8 @@ def _get_pix_coords(self, idx=None, mode='center'):
         npix = copy.deepcopy(self.npix)
 
         if mode == 'edge':
-            npix[0] += 1
-            npix[1] += 1
+            npix[0][0] += 1
+            npix[1][0] += 1
```

Comment:
What changed here? Was it a bug? Add a regression test?

Reply:
geom.npix is a tuple of arrays. The current implementation does not work. 
Maybe a better implementation would be ```npix[0] = npix[0] + 1```

Reply:
If `npix[0]` is a Numpy array, then ``npix[0][0] += 1`` and ``npix[0] = npix[0] + 1`` are the same, no? So pick either implementation. If you have time, please add a short regression test. Or I can also add it before I merge this in.

Reply:
This does not work either (tuple is immutable). Did this instead:
```           for pix_num in npix[:2]:
                pix_num += 1
```